pycbc/workflow/legacy_ihope.py

```diff
@@ -462,7 +471,6 @@ def __init__(self, cp, name, universe=None, ifo=None, injection_file=None,
                  out_dir=None, tags=[]):
         super(LegacyCohPTFInjcombiner, self).__init__(cp, name, universe,
               ifo=ifo, out_dir=out_dir, tags=tags)
-        self.cp = cp
         self.ifos = ifo
```

Comment:
As you are removing things, note that:

self.ifo_list (list of ifos (['H1','L1']))
self.ifo_string ('H1L1')

and

self.ifo (self.ifo_list if len(self.ifo_list) == 1 else raise ValueError)

are all set by the parent class, and self.ifos can be removed and replaced by one of these where needed.